addons/website/static/src/scss/website.scss

```diff
@@ -1384,6 +1396,18 @@ header {
             border-color: currentColor;
         }
     }
+
+    // On the "/Help" page (with "website_helpdesk" installed), the navbar for
+    // helpdesk teams uses the both "navbar-nav" and "nav-tabs" BS classes on
+    // the same element, which is not compatible. This results in the text and
+    // background of the active "nav-link" having the same color if the
+    // background of the website is black, and consequently, the text is not
+    // visible.
+    // TODO: In Master, modify the layout of the navbar for helpdesk teams to
+    // avoid using BS Tabs within a BS Navbar.
+    &.nav-tabs > .nav-item.active > .nav-link.active {
```

Comment:
Is there actually an active class on both item and link? Strange :shrug:

Reply:
Yes
![image](https://github.com/odoo/odoo/assets/52911687/424cf049-5d57-45b0-83ac-52697b1a240e)